propertysuggester/test/parser/test_json_reader.py

```diff
@@ -1,16 +1,12 @@
-import unittest
 import gzip
 
 from pkg_resources import resource_filename
-from nose.tools import eq_
 
-from propertysuggester.test.parser.test_abstract_reader import AbstractUniverseTest
 from propertysuggester.parser import JsonReader
 from propertysuggester.utils.datamodel import Claim, Snak, Entity
 
 
-class JsonReaderTest(AbstractUniverseTest):
-
+class TestJsonReader():
```

Comment:
Isn't this one now missing tests that used to be part of the AbstractUniverseTest? Seems to me they've only been coped to CSV and XML reader tests.